lib/connection/connection.js

```diff
@@ -193,21 +193,34 @@ class Connection extends EventEmitter {
    * Destroy connection
    * @method
    */
-  destroy() {
+  destroy(options, callback) {
```

Comment:
Is a synchronous error possible at all here? If so, we should catch it and either pass it to the callback or rethrow.

Reply:
by a call to `this.socket.destroy()` you mean? I don't think so (don't see anything in the docs for it).